src/main/java/bio/terra/service/dataset/DataDeletionRequestValidator.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.service.dataset;
+
+import bio.terra.model.DataDeletionRequest;
+import bio.terra.model.DataDeletionTableModel;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import javax.validation.constraints.NotNull;
+
+@Component
+public class DataDeletionRequestValidator implements Validator {
+
+    @Override
+    public boolean supports(Class<?> clazz) {
+        return true;
+    }
+
+    private void validateFileSpec(DataDeletionTableModel fileSpec, Errors errors) {
+        String tableName = fileSpec.getTableName();
+        if (tableName == null || tableName.equals("")) {
+            errors.rejectValue("tables.tableName", "TableNameMissing",
+                "Requires a table name");
+        }
+    }
```

Comment:
BigQuery supports at most 1 wildcard `*` in GCS references, would it be worth adding that check here?

Reply:
Yes, I want to re-use the logic that we already have for gcs-path-with-wildcard validation.